src/bindings/c/tests/ov_remote_context_test.cpp

```diff
@@ -0,0 +1,637 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#ifdef HAVE_OCL_SUPPORT
```

Comment:
Maybe we can remove thes `#ifdef`s and simple exclude this file from test binary in case if OpenCL wasn't found.

Reply:
Good idea! Will do it!

Reply:
done